src/python/foglamp/configuration_manager.py

```diff
@@ -56,29 +60,37 @@
                 value_val - string (dynamic)
 """
 
+def _run_callbacks(category_name):
+    callbacks = _registered_interests.get(category_name)
+    if callbacks is not None:
+        for callback in callbacks:
+            cb = import_module(callback)
+            cb.run(category_name)
```

Comment:
should we add:
```
except AttributeError:
              _logger.exception("{} must have implemented the run configuration-callback method".format(cb.__name__))
```

Reply:
what if this callback method wants to await another async method (may be read new configuration of the interested category-name from storage) . Basically what if we make `run` itself  async

Reply:
The callback wouldn't necessarily read back configuration. It would signal the interested component (which would read the configuration. However, signaling the interested component/microservice may be async. @MarkRiddoch what do you think?

Reply:
Added exception handling, good call @praveen-garg